pkg/bicopy/bicopy.go

```diff
@@ -1,81 +1,44 @@
-// From https://raw.githubusercontent.com/norouter/norouter/v0.6.5/pkg/agent/bicopy/bicopy.go
-/*
-   Copyright (C) NoRouter authors.
-
-   Copyright (C) libnetwork authors.
-
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
-*/
-
 package bicopy
 
 import (
+	"errors"
 	"io"
 	"sync"
-
-	"github.com/sirupsen/logrus"
 )
 
-// Bicopy is from https://github.com/rootless-containers/rootlesskit/blob/v0.10.1/pkg/port/builtin/parent/tcp/tcp.go#L73-L104
-// (originally from libnetwork, Apache License 2.0).
```

Comment:
This reference seems now lost

Reply:
Yeah, intentionally so. There's nothing left from the old implementation apart from the name (and the name `broker`).